presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java

```diff
@@ -54,7 +54,6 @@
 import static com.facebook.presto.cost.PlanNodeStatsEstimateMath.differenceInNonRangeStats;
```

Comment:
It looks like inlining `zero` is error prone, see number of places where you missed NaN